api/src/contexts/mail/usecases/IOrderCompletedInteractor.ts

```diff
@@ -0,0 +1,3 @@
+export interface IOrderCompletedInteractor {
+  OrderCompleted(orderId: string): Promise<void>;
```

Comment:
interctorにはnumber型で渡したい
```suggestion
  OrderCompleted(orderId: number): Promise<void>;
```